.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "thirdparty/basalt-mirror"]
+	path = basalt_runner/basalt-mirror
+	url = https://github.com/VladyslavUsenko/basalt-mirror.git
+[submodule "basalt_runner/jsonl-recorder"]
```

Comment:
This could be placed somehere else... maybe just jsonl-recorder in the root if there's no suitable common folder